src/sql/platform/connection/common/connectionManagementService.ts

```diff
@@ -122,11 +122,6 @@ export class ConnectionManagementService extends Disposable implements IConnecti
 
 		this._register(this._onAddConnectionProfile);
 		this._register(this._onDeleteConnectionProfile);
-
-		// Refresh editor titles when connections start/end/change to ensure tabs are colored correctly
-		this.onConnectionChanged(() => this.refreshEditorTitles());
```

Comment:
the refreshEditorTitles in the editor part was commented out and not doing anything, so I just removed the all function chain.